package.json

```diff
@@ -66,31 +67,27 @@
 			"tsx"
 		],
 		"rules": {
+			"import/no-unresolved": "off",
 			"import/no-unassigned-import": "off",
-			"no-unused-vars": [
-				2,
+			"unicorn/no-abusive-eslint-disable": "off",
```

Comment:
I liked this one

Reply:
It was just disabled while waiting for https://github.com/sindresorhus/eslint-plugin-unicorn/commit/fcee1c61a75fdf5ba35ee0da953c9ca96c1855d4

I've removed it now: https://github.com/sindresorhus/refined-github/pull/1740/commits/a1c5123aa88bc48856df0f0bec9d6348dd42200c